tests/test_views.py

```diff
@@ -1013,6 +1015,20 @@ def _setup_enrollment_client(self, client_mock):
         client.get_course_modes.return_value = self.dummy_demo_course_modes
         client.get_course_enrollment.return_value = None
 
+    def _setup_ecommerce_client(self, client_mock):
```

Comment:
@replaceafill No, not really. At some point, the code in that PR was using a weird hack (`DUMMY_PLACEHOLDER`) to include custom *logic* in a helper method. There's no custom logic here, just custom *data*. By allowing client code to pass in different values for `total`, we reduce the amount of duplication in the test code without adding any weird hacks.